sdk/attestation/azure-security-attestation/src/main/java/com/azure/security/attestation/AttestationAdministrationAsyncClient.java

```diff
@@ -190,6 +190,7 @@
     Mono<Response<String>> getAttestationPolicyWithResponse(AttestationType attestationType, AttestationTokenValidationOptions validationOptions, Context context) {
         final AttestationTokenValidationOptions validationOptionsToUse = (validationOptions != null ? validationOptions : this.tokenValidationOptions);
         return this.policyImpl.getWithResponseAsync(attestationType, context)
+            .onErrorMap(Utilities::mapException)
```

Comment:
We traditionally put OnErrorMap after flatMap in the reactor chain.

Reply:
I tried it, but it didn't compile. The only way I could get it to compile was by putting onErrorMap first. Possibly because of the type transformations done in the flatMap.

Reply:
Ohh, missed that, thanks.